src/lib.rs

```diff
@@ -22,7 +22,8 @@
 //! Logically, this crate includes these major parts:
 //! * GC components:
 //!   * [Allocators](util/alloc/allocator/trait.Allocator.html): handlers of allocation requests which allocate objects to the bound space.
-//!   * [Spaces](policy/space/trait.Space.html): memory regions in the heap with defined semantics and behaviors.
+//!   * [Policies](policy/space/trait.Space.html): definitions of semantics and behaviors for memory regions.
+//!      Each space is an instance of a policy, and takes up a unique proportion of the heap.
```

Comment:
I don't understand the meaning of _unique proportion_ here.
Does it mean two spaces can not have the same proportion of the heap (e.g. have the same size), or that spaces occupy non-overlapping virtual memory address ranges?

Reply:
It means spaces are non-overlapping in the heap.

Reply:
Not sure, but I think it's vague.
Maybe write it as:
Each space is an instance of a policy. Spaces take up non-overlapping pieces (chunks? pages? ...) of the heap.

Reply:
Yeah. Sounds good. I will change it.